tls/s2n_tls13_handshake.c

```diff
@@ -17,6 +17,19 @@
 #include "tls/s2n_cipher_suites.h"
 #include "tls/s2n_security_policies.h"
 
+static int s2n_zero_sequence_number(struct s2n_connection *conn, s2n_mode mode)
+{
+    notnull_check(conn);
+    struct s2n_blob sequence_number;
+    if (mode == S2N_CLIENT) {
+        GUARD(s2n_blob_init(&sequence_number, conn->secure.client_sequence_number, sizeof(conn->secure.client_sequence_number)));
+    } else {
+        GUARD(s2n_blob_init(&sequence_number, conn->secure.server_sequence_number, sizeof(conn->secure.server_sequence_number)));
+    }
+    GUARD(s2n_blob_zero(&sequence_number));
```

Comment:
I really wish C could zero an array without having to specify the length:

```c

    if (mode == S2N_CLIENT) {
        conn->secure.client_sequence_number = { 0 };
    } else {
        conn->secure.server_sequence_number = { 0 };
    }
```

For my own knowledge: why isn't the sequence number a `uint64_t`? That would certainly make dealing with it easier in a lot of these kinds of things.

Reply:
Good question. I don't see any obvious reason the sequence number couldn't be a uint64_t.